docs/source/conf.py

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 #
-# GeoStatTools documentation build configuration file, created by
-# sphinx-quickstart on Fri Jan  5 11:25:33 2018.
+# AnaFlow documentation build configuration file, created by
```

Comment:
Copypasta

Reply:
Sorry for that again. But that means even in the copied conf.py was a copy-mistake